src/collection.js

```diff
@@ -242,6 +242,15 @@ export default class Collection {
   }
 
   /**
+   * Closes any current connection to the database for this collection.
+   *
+   * @return {Promise}
+   */
+  close() {
+    return this.db.close();
+  }
+
+  /**
```

Comment:
Also, forcing "opening" a collection would break existing apps using Kinto.js 1.0, because you'd then need to do:

```js
const coll = kinto.collection("foo"); 
coll.open().then(_ => coll.create());
```